src/lib/util/Config.ts

```diff
@@ -22,6 +25,9 @@ export type EngineConfigContent = {
 }
 
 const DEFAULT_CONFIG: ConfigContent = {
+	// It's typically bad practice to use `require` instead of `import`, but the former is much more straightforward
```

Comment:
In order to use the recommended `await import()` syntax, we'd need to make changes to our `tsconfig.json`, and I'd want to do some digging to see what the possible consequences of that might be. In the meantime, `require()` isn't technically *wrong*, so I'm just keeping it.